fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/api/event/registry/DynamicRegistries.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright 2016, 2017, 2018, 2019 FabricMC
+ * Copyright 2023 The Quilt Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.event.registry;
+
+import java.util.List;
+
+import com.mojang.serialization.Codec;
+import org.jetbrains.annotations.Unmodifiable;
+
+import net.minecraft.registry.Registry;
+import net.minecraft.registry.RegistryKey;
+import net.minecraft.registry.RegistryLoader;
+
+import net.fabricmc.fabric.impl.registry.sync.DynamicRegistriesImpl;
+
+/**
+ *
+ * @deprecated Use Quilt Registry API's {@link org.quiltmc.qsl.registry.api.dynamic.DynamicMetaRegistry} instead.
+ *
+ * <p>Contains methods for registering and accessing dynamic {@linkplain Registry registries}.
```

Comment:
Actually, instead of doing this (which breaks Fabric's checkstyle check):
```suggestion
 * Contains methods for registering and accessing dynamic {@linkplain Registry registries}.
```